src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -1202,10 +1202,39 @@ func getTempUrls(securityScopedUrls: [URL]) -> [URL] {
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Specify a bucket
+
+We also have the ability to target a specific bucket by passing the bucket option to the `uploadData` request. The bucket option can either be A string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.. 
+
+See [Configure additional storage buckets](/[platform]/build-a-backend/storage/set-up-storage/#configure-additional-storage-buckets) for more information.
```

Comment:
I think we can remove this since the info provided is already in the table below

Reply:
I added this suggestion.